src/registrar/admin.py

```diff
@@ -58,10 +95,27 @@ def history_view(self, request, object_id, extra_context=None):
             )
         )
 
+    def formfield_for_manytomany(self, db_field, request, **kwargs):
+        """customize the behavior of formfields with manytomany relationships.  the customized
+        behavior includes sorting of objects in lists as well as customizing helper text"""
+        queryset = AdminSortFields.get_queryset(db_field)
```

Comment:
@neilmb I agree this is not ideal.  We need to call this method from both AuditedAdmin and DomainInformationInline.  So I defined a separate class for defining the method.  

It was a bit challenging to define this cleanly because multiple classes need to extend either (or both) of formfield_for_manytomany and formfield_for_foreignkey (defined in BaseModelAdmin).  While most of these classes inherit AuditedAdmin, DomainInformationInline is not able to inherit AuditedAdmin.  And so I defined a separate class which only handles the field specific logic.

I am not entirely satisfied with how I implemented this, but also can't see how to implement your suggestion.